### DISCUSSION TITLE
FB-7582: "Übersicht" bleibt weiß

### DISCUSSION THREAD

**Discussion by dev8765** [Freetz-NG/freetz-ng#994] [Allgemein]
Firmware: 156.07.17 rev107895, Branding: avme, Freetz: ng-23709M-ee7a33cbc

Bei Klick auf "Übersicht" bleibt der Content-Bereich weiß.
Der Request http://192.168.178.1/data.lua liefert nur wenige Werte:
In _Chrome_: 
`{"data":[],"hide":{"ssoEmail":true,"shareUsb":true,"liveTv":true,"dectMoni":true,"wps":true,"wGuest":true,"dectMoniEx":true,"rss":true,"mobile":true,"wKey":true,"ssoPass":true,"dectMail":true,"dectDev":true,"chan":true,"liveImg":true},"pid":"overview","sid":"b78ed7a20a722a65"}`
In _Firefox_:
`{"data":[],"hide":[],"pid":"overview","sid":"42072ac2cbd5d895"}`

Alle anderen Punkte der linken Navigation funktionieren problemlos.

http://fritz.box geht nicht, vermutlich wegen Unverträglichkeit zwischen _Win10_ und _dnsmasq_.

<img width="906" alt="fb-7582-uebersicht-weiss" src="https://github.com/Freetz-NG/freetz-ng/assets/30365427/79a92754-8cb5-4c21-8e2e-5002e1e02521">

**Comment by fda77** [comment]
mach alle patches im menuconfig aus und schau ob es dann auch so ist. falls nein mach nach und nach wieder ein paar an und finde heraus welches patch inkompatibel ist

**Comment by dev8765** [comment]
Ich habe alles deaktiviert in `Removal patches` und `Other patches` 
Leider keine Änderung

**Comment by aus-sua** [comment]
Ich glaube kaum, daß es das Problem löst, aber ein Versuch wäre es ja wert.
FOS-Einstellungen sichern und dann:
Mach doch einmal ein "Werkseinstellungen zurücksetzen" über das AVM-WebIF.

**Comment by aus-sua** [comment]
Und ggf. eben auch ohne link/tagging/Favicon testen:

![keines_](https://github.com/Freetz-NG/freetz-ng/assets/73757472/de1cd476-c59f-475d-a2fb-af52c339648a)

**Comment by dev8765** [comment]
Ich habe jetzt `Einstellungen sichern` und `Werkseinstellungen laden` gemacht
Leider keine Änderung
Interessanterweise blieben die Zugangsdaten erhalten und die Internet-Verbindung wurde wieder hergestellt ohne Restore

**Reply by aus-sua** [reply]
> Interessanterweise blieben die Zugangsdaten erhalten

so etwas hatte ich noch nie, schlägt da ein "Zugangsdaten übertragen" des ISP zu (angeblich hätte so etwas ab jetzt auch die Telekom...)?

**Reply by dev8765** [reply]
Das glaube ich nicht, die Box stammt ursprünglich von _M-net_ (München), sie hat glaube ich auch noch das Branding
Ich bin aber bei einem anderen Provider (_fonial_)

**Reply by aus-sua** [reply]
Schon komisch, schon einmal mit dem Holzhammer, mit:
[http://download.avm.de/fritzbox/fritzbox-7582/other/recover/](url)
versucht?

**Reply by aus-sua** [reply]
google sagt, fonial nutzt Telekom-Netz...
Wäre gut möglich, daß die da 'n Autokonfig machen...

**Reply by dev8765** [reply]
Damit habe ich (bei einer anderen 7582) schon schlechte Erfahrungen gemacht:
Das Recover-Programm bricht ab weil ein Branding vorhanden ist
Das Branding hatte ich dann entfernt, ich glaube auch _avme_ -> _avm_ geändert
Dann lief zwar das Recover durch
Aber die Box konnte keine Internetverbindung mehr aufbauen 
Mein ISP hat dann aufgrund meiner Fehlermeldung eine Telekom-Messung beauftragt
Ergebnis: Die Box hätte interne Fehler gemeldet, vor allem Stromversorgung
Ich habe jetzt eine andere 7582, wieder von _M-net_

**Reply by dev8765** [reply]
Ah, wenn Autoconfig vorhanden ist dann ist wenigstens verständlich dass kein Restore nötig war

**Reply by aus-sua** [reply]
Nun ja,
DSL-Stecker raus und raus lassen, 
Werkseinstellungen laden,
warten...
im FOS nachsehen, ob Zugangsdaten/ISP-Einstellungen noch da sind...

**Reply by aus-sua** [reply]
> ich glaube auch avme -> avm geändert

Das könnte damals das Problem gewesen sein, die Firmware ist ja wohl avme, die Umschaltung 
auf dt. erfolgt per Spracheinstellung.
Gut möglich, daß das auch das Problem der leeren Startseite mit freetz ist, avm gibt es so nicht, ggf. nur avme...

**Reply by dev8765** [reply]
Tatsächlich, ohne DSL-Verbindung fehlen die Daten, DSL-Stecker rein, DSL-Sync kommt, Internetverbindung kommt
Alles ohne selbst die Zugangsdaten eingegeben zu haben, schon cool...

**Reply by dev8765** [reply]
> das Problem der leeren Startseite im _freetz_ ist

Nur um sicherzugehen, _freetz_ ist nicht betroffen, es ist die _FRITZ!OS_-Startseite

**Reply by aus-sua** [reply]
irgendwie hängt github 'mal wieder etwas, ich hatte das oben schon abgeändert,
ohne Deine nachfolgende Posts angezeigt zu bekommen...

**Comment by dev8765** [comment]
Leider hat das bei mir auch nichts gebracht:
<img width="290" alt="freetz-web-interface" src="https://github.com/Freetz-NG/freetz-ng/assets/30365427/8f260edc-ac13-43d7-94e5-f88fc6aa9953">

**Comment by dev8765** [comment]
Könnte man ansetzen bei dem `POST http://fritz.box/data.lua`? 
Der bringt als Response nur 
`{"data":[],"hide":[],"pid":"overview","sid":"e312ebc4daca1a7b"}` oder 
`{"data":[],"hide":{"ssoEmail":true,"shareUsb":true,"wps":true,"liveTv":true,"dectMoni":true,"provServ":true,"wGuest":true, "dectMoniEx":true,"rss":true,"mobile":true,"wKey":true,"dectDev":true,"dectMail":true,"ssoSet":true,"chan":true,"liveImg":true},"pid":"overview","sid":"e312ebc4daca1a7b"}`.
Kann es dafür in einem Log eine Fehlermeldung geben?

**Reply by aus-sua** [reply]
Ich könnte mir vorstellen, daß das ein Problem im freetz-NG ist, das da beim Entpacken/Packen/Zusammenbau 
im AVM-WebIF entsteht, weil die AVM-Firmware avme/multilanguages ist...

**Reply by dev8765** [reply]
Wenn ich im ADAM von _avme_ -> avm ändere, kann ich das garantiert rückgängig machen?
Ich habe aktuell nur diesen 1 Router...

**Reply by aus-sua** [reply]
nun, garantieren kann Dir das keiner, 
bisher war es aber möglich, das rückgängig zu machen, ggf. dann 
eben per serieller Konsole...

**Reply by dev8765** [reply]
Wenn ich das richtig verstehe reicht die bloße Änderung nicht sondern danach muss noch das Recovery gemacht werden?
Das hat mir schon 1 Box funktionsunfähig gemacht, das möchte ich nicht, dann verzichte ich lieber auf die "Übersicht".

**Comment by dev8765** [comment]
In _Region und Sprache_ funktioniert gar nichts
Sprache auf Englisch umstellen: Das bleibt wirkungslos, auch nach Reboot
Klick auf Reiter "Ländereinstellungen": Das legt die FRITZ!OS-Webgui ganz lahm

**Comment by aus-sua** [comment]
Dein freetz-NG war für avme oder avm?
ggf. baust Du es 'mal für avme und 'mal für avm und testest da die Unterschiede?

![avme](https://github.com/Freetz-NG/freetz-ng/assets/73757472/4ab06b16-99d6-46d7-898d-41044a4c54b4)

Ich glaube aber, das ist nicht das Problem, sondern die Anpassungen für das AVM-WebIf sind nicht im freetz-NG.
Das sollte dann ja auch die 7581 betreffen...
Die 7581/82 sind leider auch bei AVM etwas "selten/anders", da sind die Rückmeldungen und Mitarbeiten 
wohl eher gering/nicht vorhanden.
Die 7369 war ja auch so eine "seltene" Box, da ist wohl das Problem ähnlich (oder wurde es da vollständig gelöst?).

**Reply by dev8765** [reply]
> Dein freetz-NG war für avme oder avm?

Bisherige Images: Im "menuconfig" war eingestellt "avm" (im freetz-WebIF wurde in "Status" angezeigt "Branding: avme")
Jetzt habe ich neu gebaut mit "Target region (avm**e** - international)": Leider keine Änderung

**Comment by aus-sua** [comment]
Der einfachste Weg wäre wohl:
Es wäre gut, wenn es jemand gäbe, der 'mal ein freetz-NG für eine 7581 erstellt,
es dort aufspielt und testet, ob da das Problem mit den weißen Seiten auch so ist...
Wenn nicht (es gab wohl schon Rückmeldungen zur 7581 dazu?), dann kann man sich wohl das 
Entpacken/Packen/Zusammenbauen des freetz-NG-Image dort von der 7581 für die 
7582 "abschauen", so viel Unterschiede sollte es da dann nicht geben.
Oder Du kämpfst Dich durch die Image-Erstellung des freetz-NG und vergleichst dort die
7581 mit der 7582...
Aber ohne den Boxen wird es wohl arg kompliziert, testen muß man es ja darauf...
